httpheader.xml

```diff
@@ -103,7 +111,7 @@
 					description="PLG_SYSTEM_HTTPHEADER_HSTS_MAXAGE_DESC"
 					default="31536000"
 					filter="integer"
-					validate="options"
+					validate="number"
```

Comment:
do you need the validate when you already have the filter? dont they both do the same thing?

Reply:
validate and filters do different things ;)